open_lm/activations.py

```diff
@@ -0,0 +1,52 @@
+import torch
+import triton
+import triton.language as tl
+
+# Forward Triton kernel
+@triton.jit
+def relu_squared_kernel(X, Y, N, BLOCK_SIZE : tl.constexpr =1024):
+    idx = tl.program_id(0) * BLOCK_SIZE + tl.arange(0, BLOCK_SIZE)
+    mask = idx < N
+    x = tl.load(X + idx, mask=mask, other=0.0)
+    y = tl.where(x > 0, x*x, 0.0)
+    tl.store(Y + idx, y, mask=mask)
+
+# Backward Triton kernel
+@triton.jit
+def relu_squared_backward_kernel(X, dY, dX, N, BLOCK_SIZE: tl.constexpr=1024):
+    idx = tl.program_id(0) * BLOCK_SIZE + tl.arange(0, BLOCK_SIZE)
+    mask = idx < N
+    x = tl.load(X + idx, mask=mask, other=0.0)
+    dy = tl.load(dY + idx, mask=mask, other=0.0)
+    dx = tl.where(x > 0, 2 * x * dy, 0.0)
+    tl.store(dX + idx, dx, mask=mask)
+
+# Custom autograd function
+class SquaredReLUFunction(torch.autograd.Function):
```

Comment:
Can you add a comment here or in relu_squared_kernel showing the torch code that this is trying to triton-ize? (presumably, torch.nn.ReLU()(x)**2)